### ISSUE TITLE
Hub fails WCAG SC 1.4.1: links are only distinguished with color

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy-hub#2075]
underlining the links would fix this quite easily
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html
